### PR TITLE
docs update on kubelet authn authz

### PR DESCRIPTION
**Pull request by timfeirg**

### CODE REVIEW

File: docs/zh_cn/administration/going-production.md
Comment (zwwhdls) [inline docs/zh_cn/administration/going-production.md]: 是 watch list apiserver

**Comment by timfeirg** [comment]
review for correctness @Hexilee

File: docs/zh_cn/administration/going-production.md
Comment (Hexilee) [inline docs/zh_cn/administration/going-production.md]: 不是很书面用语，建议修改为：
```suggestion
需要注意，就算使用了 v0.21.0 及之后的版本，CSI 驱动需要配置 `podInfoOnMount: true`，上边提到的避免报错的特性才会真正生效。如果你采用 [Helm 安装方式](../getting_started.md#helm)，则 `podInfoOnMount` 默认开启无需配置，该特性会随着升级自动启用。而如果你使用 kubectl 直接安装，你需要为 `k8s.yaml` 添加如下配置：
```

**Review by Hexilee — APPROVED** [review]

**Comment by timfeirg** [comment]
all done, waiting for final approval & merge @zwwhdls

**Review by zwwhdls — APPROVED** [review]

**Pull request merged**
